openupgrade_scripts/scripts/l10n_fr_account/18.0.2.2/pre-migration.py

```diff
@@ -14,11 +14,12 @@
 
 @openupgrade.migrate()
 def migrate(env, version):
-    openupgrade.rename_models(env.cr, model_renames)
-    openupgrade.rename_tables(env.cr, table_renames)
-    openupgrade.delete_records_safely_by_xml_id(
-        env,
-        [
-            "l10n_fr_fec.account_fr_fec_rule",
-        ],
-    )
+    if openupgrade.table_exists(env.cr, "account_fr_fec"):
+        openupgrade.rename_models(env.cr, model_renames)
+        openupgrade.rename_tables(env.cr, table_renames)
+        openupgrade.delete_records_safely_by_xml_id(
+            env,
+            [
+                "l10n_fr_fec.account_fr_fec_rule",
+            ],
```

Comment:
Let's not add lines by the formatter (the key is to not add trailing commas):

```suggestion
            env, ["l10n_fr_fec.account_fr_fec_rule"]
```

Reply:
That's pure nitpicking 😅

Reply:
Yeah, I agree, but as we are changing the lines...